src/components/organisms/Header/Header.tsx

```diff
@@ -0,0 +1,15 @@
+import { Col, Grid } from '@mantine/core';
+
+export interface HeaderProps {
+  leftButton: React.ReactNode;
+  breadcrumbs: React.ReactNode;
+}
+
+export const Header = ({ leftButton, breadcrumbs }: HeaderProps) => {
+  return (
+    <Grid>
+      <Col span={10}>{breadcrumbs}</Col>
+      <Col span={2}>{leftButton}</Col>
+    </Grid>
```

Comment:
this part is also causing this overflow issue which u can see [here](https://6315924b26061e4ff69d1b5b-looeujezdb.chromatic.com/?path=/story/app--default)
I fixed it in my pr u can take look at there

Reply:
or I'll fix it after u just merge it